/home/duckietown/scm/duckuments/deploy/local/lib/python2.7/site-packages/ruamel.ordereddict-0.4.9-py2.7-linux-x86_64.egg/_ordereddict.py:3: UserWarning: Module _ruamel_yaml was already imported from None, but /home/duckietown/scm/duckuments/deploy/lib/python2.7/site-packages/ruamel.yaml-0.15.16-py2.7-linux-x86_64.egg is being added to sys.path
           mcdp|note_errors_inline.py:36   - <module>       | [31m[31mFound a few errors:[0m[0m
           mcdp|note_errors_inline.py:37   - <module>       | [31m[31m

* syntax error
  
  This link is invalid:
  
      URL = #https://photos.app.goo.gl/AirDLHRXUiImuX7x1
  
  I think there is an extra "#" at the beginning.
  
  Note that the Markdown syntax is:
  
      [description](URL)
  
  where URL can be:
  
      1) using the fragment notation, such as
  
          URL = '#SECTIONID'
  
      for example:
  
          Look at [the section](#section-name)
  
  
      2) a regular URL, such as:
  
          URL = 'http://google.com'
  
      that is:
  
          Look at [the website](http://google.com)
  
  
  You have mixed the two syntaxes.
  
  You probably meant to write the url
  
      https://photos.app.goo.gl/AirDLHRXUiImuX7x1
  
  but you added an extra "#" at the beginning that should have not been there.
  
  Please remove the "#".
  
  [0m[0m
